Formula/molten-vk.rb

```diff
@@ -0,0 +1,107 @@
+class MoltenVk < Formula
+  desc "Implementation of the Vulkan graphics and compute API on top of Metal"
+  homepage "https://github.com/KhronosGroup/MoltenVK"
+  url "https://github.com/KhronosGroup/MoltenVK/archive/v1.0.15.tar.gz"
+  sha256 "32b3942722b9feb8250a88d44a018c08f53816f2f895b6199b4b8c2e812d8fc2"
+
+  depends_on "cmake" => :build
+  depends_on :macos => :sierra # requires IOSurface/IOSurfaceRef.h
+
+  # MoltenVK depends on very specific revisions of its dependencies.
+  # For each resource the path to the file describing the expected
+  # revision is listed.
+  resource "cereal" do
+    # ExternalRevisions/cereal_repo_revision
+    url "https://github.com/USCiLab/cereal/archive/51cbda5f30e56c801c07fe3d3aba5d7fb9e6cca4.tar.gz"
```

Comment:
should be
```
url "https://github.com/USCiLab/cereal.git",
    :revision => "51cbda5f30e56c801c07fe3d3aba5d7fb9e6cca4"
```
and the same for the others

Reply:
Thanks, fixed!